UnityProject/Packages/manifest.json

```diff
@@ -21,7 +21,7 @@
     "com.unity.ugui": "2.0.0",
     "com.unity.vectorgraphics": "2.0.0-preview.24",
     "com.whisper.unity": "https://github.com/Macoron/whisper.unity.git?path=/Packages/com.whisper.unity",
-    "org.unitystation.spacemirror": "https://github.com/unitystation/Mirror.git?path=/Assets/Mirror#updatingv2",
+    "org.unitystation.spacemirror": "https://github.com/unitystation/Mirror.git?path=/Assets/Mirror#upstream-only",
```

Comment:
why is it on a different branch?

Reply:
so we can return to the old one when needed.